### ISSUE TITLE
Bootstrap 4 Update

### ISSUE THREAD

**Issue by MikeyUchiha** [cloudscribe/cloudscribe#356]
Hi Joe,

Bootstrap 4 has been officially released as of last month.

http://blog.getbootstrap.com/2018/01/18/bootstrap-4/

What do you think the timeframe would be to get the Nuget Packages added for it?

Thanks

**Comment by joeaudette** [comment]
I don't know how that announcement escaped my attention but that is great news! Work on this will begin soon.

**Comment by MikeyUchiha** [comment]
I was surprised as well. I only found out because a friend had told me he was using it for one of his client’s projects.

**Comment by joeaudette** [comment]
Working on views for bootstrap4, making note of some issues encountered.

in bootstrap3 and in bootstrap4 [nested dropdowns are notsupported](https://github.com/twbs/bootstrap/issues/21026)

we worked around the issue in bootstrap3 and will have to workaround it in bootstrap4 since a cms can have any number of nested levels of pages.

It seems like they made it a little harder in bootstrap4 because the sub menu dropdown uses divs instead of nested ul or ol. it would be better semantics for navigation to use nested lists.

I found this which I will try to use or adapt
https://github.com/bootstrapthemesco/bootstrap-4-multi-dropdown-navbar
